database/buildSchema/34B_review_assignable_functions.sql

```diff
@@ -0,0 +1,143 @@
+-- Function to return elements reviewable questions (per application)
+CREATE OR REPLACE FUNCTION public.reviewable_questions (app_id int)
+    RETURNS TABLE (
+        code varchar,
+        response_id int,
+        is_reviewable public.is_reviewable_status,
+        response_value jsonb,
+        is_optional boolean
+    )
+    AS $$
+    SELECT DISTINCT ON (code)
+        te.code AS code,
+        ar.id AS response_id,
+        te.is_reviewable AS is_reviewable,
```

Comment:
Just a thought -- perhaps we should change the name of the "is_reviewable" field to "reviewability", as it's now slightly misleading as it has several states?

Might be a hassle though, only do it if it's not too messy.

Reply:
I'll have a go now

Reply:
It was ok to do - just needed to be done before other changes on `migrationData` code